sdk/devcenter/azure-developer-devcenter/tests/test_smoke.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+import os
+from testcase import DevcenterTest, DevcenterPowerShellPreparer
+from devtools_testutils import recorded_by_proxy
+
+class TestDevcenterSmoke(DevcenterTest):
+    @DevcenterPowerShellPreparer()
+    @recorded_by_proxy
+    def test_smoke(self):
+        dev_center = "sdk-default-dc"
+        tenant_id = "88888888-8888-8888-8888-888888888888"
+        client = self.create_client(tenant_id=tenant_id, dev_center=dev_center)
```

Comment:
@chrissmiller  You can use `devcenter_endpoint` directly like: 
```
def test_smoke(self, devcenter_endpoint):
```
as long as it is defined in `DevcenterPowerShellPreparer`